docs/PacketFence_Developers_Guide.asciidoc

```diff
@@ -118,12 +118,44 @@ HTTP JSON API
 
 PacketFence provides an HTTP JSON API which exposes most of its functionality. 
 
-The full API spec documentation for the latest stable release is available from: http://petstore.swagger.io/?url=https://raw.githubusercontent.com/inverse-inc/packetfence/stable/docs/api/spec/openapi.yaml 
+The full API spec documentation for :
+
+* the latest stable release is available from: https://packetfence.org/doc/api/
+* the latest devel release is available from: https://packetfence.org/doc/api-devel/
 
 The API is exposed on 127.0.0.1 (localhost) as well as the management interface of the server for remote calls.
+You should use 9999 TCP port and a secure connection (HTTPS) to ask it.
 
 Authentication can be done using either the webservices credentials or any credentials that are valid on the web admin interface.
 
+How to use API
+~~~~~~~~~~~~~~
+
+NOTE: You can use API spec documentation mentionned above to have ready to use curl commands with "Try it out" button.
```

Comment:
*NOTE: The API spec documentation mentioned above provides ready-to-use curl commands through the "Try it out" button.*